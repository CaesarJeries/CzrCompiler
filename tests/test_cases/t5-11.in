int zero() { return 0; }
byte zeroB() { return 0b; }
int printAndRet() {
    print("\ncalled printAndRet\n");
    return 1;
}

void main() {
    printi(zero());
    print("\n");
    printi(zeroB());
    print("\n");
    printi(zero() + 1 + 2 + 3 + 4 + 5 + 6 + 7 + 8 + 9 + 10 + 11 + 12 + 13 + zeroB());
    printAndRet();
    printi(zero() + zero() + zeroB());
    printi(printAndRet());
}