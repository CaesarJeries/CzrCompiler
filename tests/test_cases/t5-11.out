Loaded: ./exceptions.s
0
0
91
called printAndRet
0
called printAndRet
1